src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/Program.cs

```diff
@@ -83,3 +91,89 @@
 app.MapControllers();
 
 app.Run();
+
+
+public class SnakeCaseNamingPolicy : JsonNamingPolicy
```

Comment:
TODO: move to common library. Last minute change when reviewing the Arc service with Burak.